qubes_menu/appmenu.py

```diff
@@ -144,6 +147,15 @@ def _add_cli_options(self):
             None,
         )
 
+        self.add_main_option(
```

Comment:
there's already a feature for that, this duplicates existing functionality (see https://github.com/QubesOS/qubes-desktop-linux-menu/pull/50 )

Reply:
If you think this is sensible, I think it should be documented how does this interact with the feature (at least in this option's help)